### DISCUSSION TITLE
Xaml Islands state WINUI + wpf

### DISCUSSION THREAD

**Discussion by rmarinho** [microsoft/WindowsAppSDK#2262] [Q&A]
IS there a experimental package to try WINUI3 with wpf app ?

**Comment by ghost** [comment]
**WinUI 3 XI** (xaml island)  is not ready. You could use WinUI 2 XI as that's what Windows 11 uses all over the place. (new taskmgr , File Explorer and so on).
see https://github.com/microsoft/WindowsAppSDK/discussions/2233#discussioncomment-2338216 and upvote in their roadmap https://portal.productboard.com/winappsdk/1-windows-app-sdk/c/9-xaml-islands-use-winui-3-from-wpf-or-winforms

and try to use the github search  function.
